src/widgets/ImageRoll.cpp

```diff
@@ -421,41 +421,3 @@ void ImageRoll::Draw(wxDC &dc, wxRect rect, int WXUNUSED(logicalFunc))
 
    } // switch
 }
-
-BEGIN_EVENT_TABLE(ImageRollPanel, wxPanelWrapper)
```

Comment:
The changes in commit "ImageRollPanel usage replaced with wxPanelWrapper" are not necessary for this issue.  Not harmful either, as they remove and change two unused classes.  Why did you think you needed this?

Reply:
For same reason we are trying to avoid dependency cycles, because aliases complicates the code